src/main.ts

```diff
@@ -23,6 +23,7 @@ async function run(): Promise<void> {
       branch: core.getInput("branch"),
       directory: core.getInput("directory"),
       syncAction: core.getInput("sync_action") as SyncAction,
+      localizedStrings: core.getInput("localized_strings"),
```

Comment:
Could you rename it to `localized_strings_path` and include `localized_strings_format` optional arg for the format?

This is because the `lingui` format also uses the `json` file extension. We could maybe fallback to the file extension if no format is provided (so there would be no need to specify `po` format if the file extension is `.po`)

Reply:
renamed and added format... like before, went with localization_strings_format to match sync command but can changed to localized if you prefer 

arg name: https://github.com/plasmicapp/plasmic-action/pull/3/commits/7e61af0df6cd55eeed2370b5bb7031ec393b4d79#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR28

format vs ext logic: https://github.com/plasmicapp/plasmic-action/pull/3/commits/72759425822c7d476b95a102b1f1da7b1478ee01#diff-ccec8a6cd159796cbca5166b919d75afa7e362394ac31ed3ef472ac5af0608daR141